frontend/src/app/forms/forms-routing.module.ts

```diff
@@ -0,0 +1,27 @@
+/*
+@Author
+Bharathwaj Ravi
+
+Add modifiers under @Modifier
+@Modifiers
+
+@Purpose
+  - This page has the route path configuration in forms module.
+*/
```

Comment:
Please remove this header. It is really uncommon to have a header like this in a TS project. Also, since we are all working on the project, you are not the only Author of this file.

(Also, it is self-exmpanatory that this page contains routes.)

I will not comment on the other files, but please remove these headers there as well.

Reply:
I feel these comments should be there to give a fair overview of what's the component used for, info regarding Author so that someone can contact if they have doubts, Also Modifiers name to just to see who made the changes without seeing file history. With my previous experience, I witnessed it was useful and hence added it.

Reply:
We have collective code ownership. So it makes no sense to say "x is the author". Also, we are using git, so you can see who did what anyways. Also: If 10 people edit the file - should they all be listed here?

If you really want to use the Modifiers, then at least write them as lowercase. (As, for instance, done in the Angular core https://github.com/angular/angular/blob/master/packages/common/src/common.ts )
But again - you usually just want to use plain comments and some built-in ones (like `@param`) that have Intellisense support.